src/main/java/reiff/dictionary/DictionaryFrame.java

```diff
@@ -0,0 +1,75 @@
+package reiff.dictionary;
+
+import com.opencsv.exceptions.CsvValidationException;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.io.IOException;
+import java.util.List;
+
+public class DictionaryFrame extends JFrame {
+    private final JTextField wordField;
+    private final JTextArea definitionsArea;
+    private EnglishDictionary dictionary;
+
+    public DictionaryFrame() {
+        try {
+            dictionary = new EnglishDictionary();
+        } catch (IOException | CsvValidationException e) {
+            e.printStackTrace();
+            JOptionPane.showMessageDialog(null, "Error loading the dictionary: " + e.getMessage(),
+                    "Error", JOptionPane.ERROR_MESSAGE);
+        }
+
+        final JFrame frame = new JFrame("DictionaryFrame");
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setSize(400, 300);
```

Comment:
Again. This isn't how I taught you to do this

Reply:
Are you referring to the order of the lines?

Reply:
I understand now, thank you!